### ISSUE TITLE
NTR: Man(5)GlcNac(2)-PP-dolichol ER flippase activity

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#28721]
Please provide as much information as you can: 

* **Suggested term label:**

Man(5)GlcNac(2)-PP-dolichol ER floppase activity

* **Definition (free text)**

Catalysis of the movement of a Man(5)GlcNac(2)-PP-dolichol from the cytosolic to the exoplasmic leaftlet of a membrane, using energy from the hydrolysis of ATP. 

* **Reference, in format PMID:#######**

PMID: 38886340

* **Gene product name and ID to be annotated to this term**
rft1

pombe https://www.pombase.org/gene/SPBC887.19
human https://www.uniprot.org/uniprotkb/Q96AA3/entry
cerevisiae https://www.yeastgenome.org/locus/S000000116

* **Parent term(s)**

GO:0140327   floppase activity

* **Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)**

M5GN2-PP-Dol flippase activity
Man(5)GlcNAc(2)-PP-dolichol translocator
Man5GlcNac2-PP-Dol ER flippase activity

* **Other**
correct spelling exoplasmic leaftlet -> leaflet in flippase and floppase

**Comment by ValWood** [comment]
Maybe that's because offically it is a floppase. Flippase seems to be used generically...

This paper calls it a"flippase"
These data, combined with in vitro results demonstrating substrate selectivity and rft1∆ phenotypes, confirm the molecular identity of Rft1 as the M5GN2-PP-Dol ER flippase.

**Comment by ValWood** [comment]
Woah thats confusing. For organelles, floppases  , from the cytosolic side (inner leaflet) to the lumenal side (outer leaflet).

So these  are floppases...

**Comment by pgaudet** [comment]
WRT flippase and floppase: @marcfeuermann and I had looked at this, and the flippase flips the molecule from the exoplasmic to the cytosolic leaftlet of a membrane, while the floppase flips from the cytosolic to the exoplasmic leaftlet of a membrane. 

Usage in the literature seems quite inconsistent.

**Comment by pgaudet** [comment]
WRT substrate: do we need such a specific substrate? I suppose this is the specificity of the transporter? Currently the human gene is annotated to GO:0034202 glycolipid floppase activity; is this not specific enough? 

Thanks, Pascale
--


[glycolipid floppase activity](https://www.ebi.ac.uk/QuickGO/term/GO:0034202)

**Comment by deustp01** [comment]
> glycolipid floppase activity

Is there a flippase vs floppase distinction to be observed here (independent of the issue of what specifically is getting flipped / flopped)?

**Comment by ValWood** [comment]
I thought we should,  because of its importance in oligosaccharide and GPI biosynthesis, and according to chat GPT, it is quite distinct  (see below).  For this reason, I did not suggest a glycolipid parent. It also took 20 years to unambiguously locate it. For a while thought it was still not located until I found this paper. confirming that it was, in fact, rft1.  Precise annotation here would have saved me a few hours because I went on a function hunt last night, and made a prediction for an unknown which I now had to retract.

Man(5)GlcNac(2)-PP-dolichol: The structure includes a lipid (dolichol) and a sugar chain (Man(5)GlcNac(2)) linked through a pyrophosphate (PP) group. While it has a lipid and sugar components, its primary role is in transferring the sugar chain to proteins or other lipid molecules rather than functioning as a glycolipid.
In summary, while Man(5)GlcNac(2)-PP-dolichol contains both lipid and carbohydrate components, it is not classified as a glycolipid because its primary function is to act as a glycosyl donor in biosynthesis rather than as a structural component of membranes.

**Comment by ValWood** [comment]
> Is there a flippase vs floppase distinction to be observed here (independent of the issue of what specifically is getting flipped / flopped)?

 You mean a directionality. I hadn't considered that. I presumed it was only moving the Man(5)GlcNac(2)-PP-dolichol into the ER...will look...

**Comment by ValWood** [comment]
I guess the experiments don't distinguish:

Rft1 protein is required for M5GN2 flipping
Rft1 is specific for M5GN2-PP-Dol translocation

but presumably, only floppase is functionally relevant?

<img width="680" alt="Screenshot 2024-08-20 at 13 50 53" src="https://github.com/user-attachments/assets/5d1a8afa-4112-4dc3-a532-8a1ed622871b">

**Comment by pgaudet** [comment]
> but presumably, only floppase is functionally relevant?

This seems right to me.

**Comment by deustp01** [comment]
> You mean a directionality.

Nothing that sophisticated - I was reading out of order and didn't realize that the flippase / floppase distinction had already come up in the thread and been handled. Sorry for the confusion.

**Comment by lbreuza** [comment]
Hi Val,
I recently updated human RTF1 in UniProt and at that time the MF was still unclear : "However, the intramembrane lipid transporter activity could not be confirmed in vitro". Once you are done with your update, please let me know. I will check what I can do from my side to better represent current knowledge for these proteins.
Thanks.

**Comment by ValWood** [comment]
Hi @lbreuza great,  this is very recent (June), so I didn't expect it to be annotated yet, I was only pointing out the usefulness of the specificity of the annotation.

The floppase I was actually looking for wasn't  this one. I got confused, I'm looking for  the floppase that transfers dolichol-P-mannose in GPI biosynthesis  if anyone knows the identity of that one. That's the one I haven't been able to locate.

**Comment by lbreuza** [comment]
But you are asking for the new term right? RTF1 is currently curated with GO:0034202 | glycolipid floppase activity by Reactome which is maybe sufficient. The term you requested seems very specific.

**Comment by ValWood** [comment]
Hi @lbreuza

Actuially you are correct. When I opened the ticket I was thinking about the "dolichol-P-mannose floppase" which isn't a glycolipid  and is still not known. 

This (rft1)  is currently the only example of a glycolipid floppase so we don't need an additional term.
Instead I will add as a synonym.
We can always add specificity  if new classes arise.

Thnaks!

**Comment by pgaudet** [comment]
Close?

**Comment by ValWood** [comment]
I wan't to add narrow synonym and fix a couple of typos I spotted, keep open and I'll do it on my next GO day
